assignment2.txt

```diff
@@ -0,0 +1,14 @@
+Questions for assignment 2
+
+1. Yes they can if they are separated by && (and) or || (or).
+
+2. A logical error will not give you an error or a bug in your code but will not make sense and cause the program to stop. The issue does not come from an error with reading the code but from the logic behind the steps.
```

Comment:
This is true! It's also important to note that syntax errors occur when you compile, whereas logical errors appear during runtime.